flight/PiOS/Common/pios_hal.c

```diff
@@ -452,6 +463,8 @@ static void PIOS_HAL_ConfigureIBus(const struct pios_usart_cfg *usart_ibus_cfg,
  * @param[in] dsm_mode Mode in which to operate DSM driver; encapsulates binding
  * @param[in] sbus_cfg SBUS configuration for this port
  */
+
+#ifdef PIOS_INCLUDE_USART
```

Comment:
Yes.  It's kinda a hack the way stuff is disabled in PIOS_HAL.  PIOS_HAL is just in the simulator right now because we need it for the speed change function that some modules use-- a lot of this is hackery to get it to compile.  I figure this can get cleaned up a little when I work on Pi I2C.